### ISSUE TITLE
Where to get 83.23 (Feb 28) version?

### ISSUE THREAD

**Issue by richardFK** [joncampbell123/dosbox-x#3308]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

Do not know where to get the recent version - the main site only has dosbox-x-win64-0.83.22-setup (for windows) which I already have.

**Comment by rderooy** [comment]
Yes, it seems the website was not updated with the new versions, and also some builds are missing for 0.83.23.
@Wengier I think normally does these updates, but I guess he has been busy elsewhere.

You can get the currently available windows versions from here in the interim: https://github.com/joncampbell123/dosbox-x/releases

**Comment by emendelson** [comment]
@richardFK You could use the Windows installer for the old versions, and then simply replace the DOSBox-X executables with the Windows versions that @rderooy linked to.

**Comment by Wengier** [comment]
@rderooy I have updated the gh-pages branch accordingly. But as you said since I have been busy elsewhere, hope someone else will be able to do it in the future.
